moleculekit/smallmol/smallmol.py

```diff
@@ -756,6 +756,102 @@ def write(self, fname, frames=None, merge=True):
     def view(self, *args, **kwargs):
         self.toMolecule().view(*args, **kwargs)
 
+    def describe(self):
+        """
+        Returns rdkit descriptors for the molecule, like DESC_NumRotatableBonds or DESC_MolLogP. See rdkit.Chem.Descriptors for more.
+
+        Returns
+        -------
+        smart: str
+            The smarts string
+        """
+        from rdkit.Chem import Descriptors
+
+        desc_dict = dict(Descriptors.descList)
+        descs = list(desc_dict.keys())
+        descs.remove('Ipc')
+        ans = {}
+        for descname in descs:
+            try:
+                desc = desc_dict[descname]
+                value = desc(self._mol)
+            except (ValueError, TypeError, ZeroDivisionError) as exception:
+                logger.warning(f"Descriptor {descname} was not calculated for this small molecule due to {str(exception)}")
+                value = 'NaN'
+
+            bin_name = 'DESC_{}'.format(descname)
```

Comment:
try to use f-strings everywhere. And black for autoformatting. `f"DESC_{descname}"`